code/ui/richpanel/RichPanel.cs

```diff
@@ -0,0 +1,12 @@
+namespace TTTReborn.UI
+{
+    public partial class RichPanel : TTTPanel
+    {
+        public RichPanel() : base()
+        {
+            StyleSheet.Load("/ui/richpanel/RichPanel.scss");
+
+            //AcceptsFocus = true;
```

Comment:
Need this comment?

Reply:
Yea as this is a next step to catch shortcuts :)